modules/tinymce/src/plugins/lists/test/ts/browser/RemoveTest.ts

```diff
@@ -1,36 +1,48 @@
-import { Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { LegacyUnit, TinyLoader } from '@ephox/mcagar';
+import { describe, it } from '@ephox/bedrock-client';
+import { LegacyUnit, TinyAssertions, TinyHooks } from '@ephox/mcagar';
+import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
 import Plugin from 'tinymce/plugins/lists/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.lists.RemoveTest', (success, failure) => {
-  const suite = LegacyUnit.createSuite<Editor>();
-
-  Plugin();
-  Theme();
+describe('browser.tinymce.plugins.lists.RemoveTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'lists',
+    add_unload_trigger: false,
+    disable_nodechange: true,
+    indent: false,
+    entities: 'raw',
+    valid_elements:
+      'li[style|class|data-custom],ol[style|class|data-custom],' +
+      'ul[style|class|data-custom],dl,dt,dd,em,strong,span,#p,div,br',
+    valid_styles: {
+      '*': 'color,font-size,font-family,background-color,font-weight,' +
+        'font-style,text-decoration,float,margin,margin-top,margin-right,' +
+        'margin-bottom,margin-left,display,position,top,left,list-style-type'
+    },
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin, Theme ]);
 
-  suite.test('TestCase-TBA: Lists: Remove UL at single LI', (editor) => {
-    editor.getBody().innerHTML = LegacyUnit.trimBrs(
+  it('TBA: Remove UL at single LI', () => {
+    const editor = hook.editor();
+    editor.setContent(
       '<ul>' +
       '<li>a</li>' +
       '</ul>'
     );
 
     editor.focus();
```

Comment:
Some more focus calls that happen on each test, so these should be moved to a `beforeEach` hook.

- [ ] Move focus calls to `beforeEach` hook